pages/docs/v3/updating/3-0.md

```diff
@@ -256,6 +256,41 @@ In `ios/App/App/AppDelegate.swift`, update the following:
      }
 ```
 
+### Remove USE_PUSH macro
```

Comment:
It's a minor technicality but, strictly speaking, it's not a macro as the Swift compiler doesn't use a preprocessor like a C compiler
```suggestion
### Remove USE_PUSH compilation condition
```